codebuild/bin/install_awslc.sh

```diff
@@ -48,7 +54,7 @@ git clone https://github.com/awslabs/aws-lc.git --branch "$AWSLC_VERSION" --dept
 install_awslc() {
 	echo "Building with shared library=$1"
 	cmake ./aws-lc -Bbuild -GNinja -DBUILD_SHARED_LIBS=$1 -DCMAKE_BUILD_TYPE=relwithdebinfo -DCMAKE_INSTALL_PREFIX="${INSTALL_DIR}" -DFIPS="${IS_FIPS}"
```

Comment:
I think it might be a bit more "cmake idiomatic" to use [CMAKE_C_COMPILER](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER.html) instead of the CC/CXX environment variables.